samples/GraphQL.DataLoader.Example/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "GraphQlPath": "/graphql",
+  "EnableMetrics": true,
```

Comment:
Metrics can have a very bad impact on performance. Would it be a good idea to enable them in a sample project?

Reply:
yeah i guess not. With the new extracted Polyfill @Shane32  set up, this wasn't used anymore anyway.